### ISSUE TITLE
False positives with arrow function shorthand

### ISSUE THREAD

**Issue by jrolfs** [playwright-community/eslint-plugin-playwright#21]
The `playwright/missing-playwright-await` rule seems to be reporting false positives for expectations that are returned from brace-less arrow functions.

```ts
{
  /* ... @state/test machine ... */
  
            unchecked: {
              meta: {
                test: async ({ page }) =>
                  expect(page).toHaveText('Terms agreement is required'), // ← false positive, even though implicitly returned
              },
            },
            
  /* ... @state/test machine ... */
}
```

I'll try to put together a simple reproduction case.
